_digest/october-2020/using-apps-and-software-to-manage-time.mdx

```diff
@@ -0,0 +1,107 @@
+places could be an agenda book, a planner, or a calendar hanging in your room. However, as technology has advanced, better and more efficient methods 
+have been created. Most of these come in the form of apps or software that you can download on your phones, computers, and even your smartwatches. These apps 
+and software can be mainly categorized into two groups: calendar apps, which keep track of your time, and reminder apps, which keep track of your tasks. However, 
+neither of these types can do everything, and because of this, the best way to manage your time and tasks is to use a combination of these. </p>
+
+<p> Calendar apps are the more popular of the two groups. They generally work by displaying events that you put on them. Many of these apps exist but some 
+of the most popular are Google Calendar, Apple Calendar, and Calendar.com. All of these apps have their own features that make them different from the others. 
+However, with all of these apps, you have the ability to schedule when different things are happening. For example, you can use these apps to schedule when you 
+have an after school class, keep track of when you’re going to do homework, keep track of when you’re hanging out with friends, etc. Also, with most of these apps, 
+you have the ability to share an event with other people for collaborative work. By doing this, you can all have the same time written on your calendars so there 
+is no confusion. If you are someone who likes to have your schedule prepared in advance, using a calendar app is a great idea. However, a major disadvantage of 
+these apps is that many of your small tasks can not be easily managed and kept track of on a calendar, for this reason, using reminder apps is also a great idea. </p>
+
+<div align="center">
+    <figure>
+        <img src="/resources/blog/october-2020/google-calendar.png" alt="Google calendar" width="500"/>
+    </figure>
+</div>
+
+<p> By using reminder apps, you can make a list of tasks or simple reminders that you want to remember. Some of the most used reminder apps are Todoist, 
+Wunderlist, and Toodledo. These apps are all essentially places where you can jot down your to-do list and set reminders for them. By doing this you can stay 
+on top of simple tasks, homework, due dates, etc. Many of these apps also have a feature where you can share your tasks with others for collaboration. 
+These apps are great for the type of people who like to have all of their tasks and work—from tiny things such as watering your plants to big things such as 
+studying for a final exam—written down in one place with reminders being sent to them often on their phone or computer. However, just like calendar apps, these 
+apps aren’t perfect. They generally lack the ability to display when you are planning to do your events. </p>
+
+<div align="center">
+    <figure>
+        <img src="/resources/blog/october-2020/todoist.png" alt="Todoist" width="500"/>
```

Comment:
You need to change this to `<img src="/blog/october-2020/todoist.png" alt="Todoist" width="500"/>`. The `/resources/` you prepended in the latest commit makes it look for images under `/resources/blog/october-2020/`, but because the image folder in `/public/` is still `/public/blog/october-2020`, the images are still stored under `/blog/october-2020`.